kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-tree/kie-pmml-models-drools-tree-tests/src/test/java/org/kie/pmml/models/drools/tree/tests/SimpleSetPredicateTreeTest.java

```diff
@@ -32,7 +32,6 @@
 import org.kie.pmml.models.tests.AbstractPMMLTest;
 
 @RunWith(Parameterized.class)
-@Ignore
```

Comment:
@danielezonca 
I think to have removed them from "tests" projects.
I tried to run "Optimize import" on whole kie-pmml-trusty module, but it modified too many classes, so I preferred to avoid that